__tests__/app.test.js

```diff
@@ -56,3 +56,22 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("400: Returns bad request if article_id is NaN", () => {
```

Comment:
Where is our 200 test?  We've written lots of code that doesn't have a corresponding test! :O